recipes/compas_nurbs/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "compas_nurbs" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3c9e9d3aba033990d4024cafc4db10148455764652dc2cde935bc8208e76c99e
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```

Comment:
Please [use grayskull](https://conda-forge.org/blog/posts/2020-03-05-grayskull/) to create the recipe. It will fix it to some modern syntax like the script line here.

Reply:
Updated it after re-generating it with `grayskull` (except the python lower bound, otherwise the linter is unhappy).